### PR TITLE
Validate Vexta Token

### PR DESCRIPTION
**Pull request by Caprimc**
# Validate [VEX](https://explorer.solana.com/address/CzNgDirzo1gaYwVrGaZmscTKTjVffhBrN3CrkQtfDKCo)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/capri117266/status/1750178568187809877?s=20
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [ ] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by Caprimc — COMMENTED** [review]
What I should do?

**Review by Caprimc — COMMENTED** [review]
I would like my token be validated. It is really important as I don't have really that much knowledge and that will really help.

**Review by Caprimc — COMMENTED** [review]
What is that mean?

File: validated-tokens.csv [hunk 1]
```diff
@@ -1077,4 +1077,5 @@ Elementerra,ELE,8A9HYfj9WAMgjxARWVCJHAeq9i8vdN9cerBmqUamDj7U,9,https://elementer
 Fluid USDC,fUSDC,Ez2zVjw85tZan1ycnJ5PywNNxR6Gm4jbXQtZKyQNu3Lv,6,https://static.fluidity.money/images/tokens/fUSDC.png,true
 Wen,WEN,WENWENvqqNya429ubCdR81ZmD69brwQaaBYY6p3LCpk,5,https://shdw-drive.genesysgo.net/GwJapVHVvfM4Mw4sWszkzywncUWuxxPd6s9VuFfXRgie/wen_logo.png,true
 PELFORT,$PELF,BgJW7U1u2RY5XJk9uYb5AqFRzjMtqE7pw3kaf9iw9Ntz,6,https://raw.githubusercontent.com/yamitora0/pelfort/main/logo.png,true
-Jupiter,JUP,JUPyiwrYJFskUPiHa7hkeR8VUtAeFoSYbKedZNsDvCN,6,https://static.jup.ag/jup/icon.png,true
```
Comment (Caprimc) [inline validated-tokens.csv:1080]: What is wrong with this?

**Comment by Jymfletzfur** [comment]
insufficient volume

**Comment by Caprimc** [comment]
> insufficient volume

What is that mean? What I should do?

**Pull request closed without merging**